helpdesk_mgmt_project/views/helpdesk_ticket_view.xml

```diff
@@ -33,6 +33,11 @@
         <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_form" />
         <field name="arch" type="xml">
             <xpath expr="//field[@name='partner_email']" position="after">
+                <field
```

Comment:
Hi thanks for this PR, could you explain the use of this ?
AFAICR, either the field is not pushed in the view if you're not in the group and if you're member of the gourp, it's invisible. 
my conclusion is that you never display nor use the field. 
Tanks for your next precisions